shippo.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.license               = 'MIT'
   spec.metadata              = { 'shippo_documentation' => 'https://goshippo.com/docs/' }
 
-  spec.add_dependency 'rest-client', '~> 1.8'
+  spec.add_dependency 'rest-client', '>= 2.0'
```

Comment:
Also, do you know which version of rest-client we should use. If so, let's use ~> to restrict the range of version.